mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -164,12 +137,13 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
      * a fair amount of reflection.
      */
     private fun validateState(initialState: S) {
-        if (state::class.visibility != KVisibility.PUBLIC) {
-            throw IllegalStateException("Your state class ${state::class.qualifiedName} must be public.")
+        if (!Modifier.isPublic(state::class.java.modifiers)) {
```

Comment:
Slight semantic change here - `internal` would not have returned true before. Maybe that's fine, but worth noting in release notes.